server/@types/prisonTypes.ts

```diff
@@ -17,4 +17,5 @@ export type Recipient = {
   prisonNumber: string
   prisonerName: string
   prisonAddress: PrisonAddress
+  barcodeValue?: string
```

Comment:
The basic idea is that once we have been through adding all the recipients, at the point of needed barcode numbers (ie. for the copy & paste route, and the PDF route), we have a service call (see below) that iterates through the recipients and calls the API for each one to give them a barcode number. Therefore the collection of recipients in the session will have the details of the prisoner and their prison, plus their barcode number